spec/core_spec.rb

```diff
@@ -6,10 +6,10 @@
   end
 
   it 'should have default value tries as 0' do
-    expect(@core.instance_variable_get(:@tries)).to eql(0)
+    @core.tries.must_equal 0
   end
```

Comment:
As a test this is the wrong thing to test except at the beginning of the game, and so there should be an initializer inside this test.  Still, this is improved because it is getting the value from the reader, not using Ruby introspection to get to it in an unnatural way.